dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -29,7 +29,7 @@
   = javascript_include_tag 'js/makerlab'
 
 -# Droplet script dependencies (all editCode - regardless of editor)
-- if use_droplet
+- if use_droplet || use_blockly
```

Comment:
I'm a little worried about changing the include order, in case some of the scripts have side effects.  This would be good cleanup to do.  Ok if I push it to a future PR?